remix/app/components/FormTextarea/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useField } from 'remix-validated-form';
+
+import Error from './error';
+import Label from './label';
+
+interface InputProps {
+  label: string;
+  name: string;
+  placeholder: string;
+  error?: string;
+}
+
+export default function FormTextarea({ label, name, placeholder }: InputProps) {
+  const { error, getInputProps } = useField(name);
+
+  return (
+    <div className="flex flex-col gap-2 px-0.5">
+      <Label htmlFor={name} label={label} />
+      <textarea
```

Comment:
The code for the components of `FormInout` and `FormTextarea` are redundant (and duplicated). How about abstracting this via a prop:

```
<FormTextInput format="short" ... /> -> Renders a text input
<FormTextInput format="long" ... /> -> Renders a textarea
```

Reply:
Yes, that would be much better.

When I'm doing this I also know that those components are duplicated from each other, I am afraid that combining `<input>` and `<textarea>` isn't allowed.

So as suggested, I will improve it.

Reply:
Improved here: [0af22fb](https://github.com/nimblehq/js-framework-benchmark/pull/120/commits/0af22fb487b5618b545e2edf90f9b9e0fe94da67)